src/Badge/Badge.tsx

```diff
@@ -14,36 +14,69 @@ const ModeColors = {
 	warning: COLORS.Warning
 };
 
+const ProgressColors = {
+	warning: COLORS.WarningBackground
+};
+
 interface IProps {
 	title: string;
 	color?: COLORS;
 	date?: string;
 	size?: string;
 	mode?: Mode;
+	progress?: Number;
 }
 
 export const Badge = (props: IProps) => {
-	const { title, color = COLORS.Green, date, size, mode } = props;
+	const { title, color = COLORS.Green, date, size, mode, progress } = props;
+	const isProgressBar = progress && !isNaN(Number(progress));
 
 	return (
 		<div
 			className={cn("kit-badge", {
 				[`kit-badge_size_${size}`]: size,
 				[`kit-badge_mode_${mode}`]: mode
 			})}
-			style={{ backgroundColor: mode ? ModeColors[mode] : color }}
+			style={{
+				backgroundColor: mode
+					? isProgressBar
+						? ProgressColors[mode]
+						: ModeColors[mode]
+					: color
+			}}
 		>
 			<span
 				className={cn("kit-badge__text", {
-					"kit-badge__text_withPoint": date
+					"kit-badge__text_withPoint": date && !isProgressBar,
+					"kit-badge__text_bold": isProgressBar
 				})}
 			>
 				{title}
 			</span>
-			{date && (
+			{isProgressBar ? (
 				<>
-					Запущен:<span className="kit-badge__date"> {date}</span>
+					<span> {`${progress}%`}</span>
```

Comment:
Можно было просто так
```suggestion
					<span> {progress}%</span>
```